google/ggrc-core
# [GGRC-7322] Add read more / read less to change proposal and change log tabs

# Issue description

Description text in Change Log tab and in Proposals tab could be too large so we need a way to Show/Hide it

# Steps to test the changes
- Create Program
- Add large description
- Navigate to Change Log tab
- There you should now see `Read more` link/button
- Add Proposal
- Navigate to Proposals tab
- There you should now see `Read more` link/button
- Open burger menu and hover over the newly created program
- You should see Description and `Read more` link/button

# Solution description

I have simply added `read-more` component to corresponding templates

# Sanity checklist




# PR Review checklist
